packages/db/src/core/integration/vite-plugin-db.ts

```diff
@@ -77,9 +77,9 @@ export function getStudioVirtualModContents({
 	return `
 import {collectionToTable, createRemoteDatabaseClient} from ${RUNTIME_IMPORT};
 
-export const db = await createRemoteDatabaseClient(${JSON.stringify(
-		appToken
-	)}, import.meta.env.ASTRO_STUDIO_REMOTE_DB_URL);
```

Comment:
Ah okay, so this would be a runtime check for the variable rather than a build time check? `getRemoteDatabaseUrl()` checks for the environment variable internally

Reply:
Yes, `import.meta.env` is preserved in SSR builds, so it would check that at runtime, then fallback to the string you injected at build-time.